src/models/commons.rs

```diff
@@ -31,6 +34,17 @@ pub enum ParamValue {
     ArrayValue(Vec<ParamValue>),
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone, Hash, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct ValueDecoded {
+    pub operation: String, //TODO use operation, usually this is received as 0 or 1 but here it is `CALL`, etc. as a string
+    pub to: String,
+    pub value: Option<u64>,
+    pub data: Option<String>,
+    #[serde(rename(deserialize = "decodedData"))] //TODO maybe we can also open an issue for this to have names consistent across structs
```

Comment:
Yes I would do so, but it will break the web clients until they switch to our service. But would love to create an issue on the transaction service to remove this todo